package.json

```diff
@@ -4,6 +4,7 @@
   "description": "REST storage and service dispatcher",
   "main": "lib/server.js",
   "scripts": {
+    "postpublish": "git tag -a \"v${npm_package_version}\" -m \"${npm_package_name}@${npm_package_version} release\" && git push upstream --tags",
```

Comment:
Perhaps it'd be safer to push just the tag we've just created, so as to avoid ridiculous situations.